clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -377,8 +368,12 @@ async function fetchCustomFidesCss(
       const data = await response.text();
 
       if (!response.ok) {
-        // eslint-disable-next-line no-console
-        console.error(
+        if (response.status === 404) {
+          fidesDebugger("No custom-fides.css found, skipping...");
+          autoRefresh = false;
+          return null;
+        }
```

Comment:
@eastandwestwind This is the part I thought @NevilleS was referring to on Standup this morning. Slightly cleaner handling of a file that's rarely present, with the added benefit of not showing up at all if we're not in DEBUG mode.